torntools/scripts/content/global/ttGlobal.js

```diff
@@ -861,3 +863,11 @@ function tradeChatPostTimer() {
 			});
 	}
 }
+
+function easterEggs() {
+	let easterEggsObserver = new MutationObserver(() => {
+		if (doc.find("[src*='competition.php?c=EasterEggs'][src*='step=eggImage'][src*='access_token=']")) alert("There is an easter egg on this page !");
```

Comment:
First src check should be a ^=, that way people won't be able to fake it on their profiles or on the forums.